### PR TITLE
Make installation easier!

### PR DESCRIPTION
**Pull request by AGulev**
image

### CODE REVIEW

File: .github/workflows/bob.yml [hunk 1]
```diff
@@ -22,9 +22,9 @@ jobs:
     name: Build
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-java@v1
-        with:
-          java-version: '11.0.2'
+      - uses: actions/setup-java@v3
+        with: { java-version: '17.0.5+8', distribution: 'temurin'}
+
```
Comment (AGulev) [inline .github/workflows/bob.yml:27]: use newer JDK

File: .github/workflows/bob.yml [hunk 2]
```diff
@@ -48,15 +48,15 @@ jobs:
   build_with_bob_macos:
     strategy:
       matrix:
-        platform: [x86_64-macos]
+        platform: [x86_64-macos, arm64-macos]
```
Comment (AGulev) [inline .github/workflows/bob.yml:51]: make sure it builds for M1

File: README.md [hunk 3]
```diff
@@ -54,11 +54,7 @@ Try the demo: https://d954mas.github.io/defold-box2d/
 __1)__ Add defold-box2d in your own project as a Defold library dependency. Open your game.project file and in the dependencies field under project add:
 https://github.com/d954mas/defold-box2d/archive/refs/tags/1.0.0.zip
 
-__2)__ Remove Defold engine box2d. If it's not removed there will be a conflict between this box2d and engine box2d.
-Open your game.project file and in the App Manifest field under Native Extension add manifest.
-
-1. Use /box2d/manifest.appmanifest, OR
-2. You can generate a manifest yourself https://britzl.github.io/manifestation/
+> :warning: This extension removes default box2d, that means you have to remove all the Collision Object components from your collections and Game Objects
```
Comment (AGulev) [inline README.md:57]: image

we don't need it anymore, see the next comment

File: box2d/native/ext.manifest [hunk 4]
```diff
@@ -1 +1,95 @@
-name: "box2d"
\ No newline at end of file
+name: "box2d"
```
Comment (AGulev) [inline box2d/native/ext.manifest:1]: We can specify everything we need to exclude or include right here. So you don't need to manually install app manifest anymore

**Comment by AGulev** [comment]
Pls don't forget to release new version 1.0.1 and fix Readme.md with updated version link!

File: box2d/native/ext.manifest [hunk 5]
```diff
@@ -1 +1,95 @@
...
+
+platforms:
+    armv7-ios:
+        context:
+            excludeLibs: [physiwcs, LinearMath, BulletDynamics, BulletCollision, Box2D]
```
Comment (vlaaad) [inline box2d/native/ext.manifest:6]: physiwcs?

Comment (AGulev) [reply box2d/native/ext.manifest]: nice one, thx!

**Pull request merged**